package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -835,12 +835,15 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
             }
           }
 
-          if (!value || value == "" || value == "null") {
-            if (!this.nullValueParams) {
-              this.nullValueParams = [];
-            }
+          if(!(value == "" && type == "TextInputComponent"))
```

Comment:
To avoid using additionally "type" input parameter we can check "param.list" property. Something like this:
```javascript
          if (param.list && (!value || value == "" || value == "null") {
            if (!this.nullValueParams) {
              this.nullValueParams = [];
            }
            this.nullValueParams.push(param);
          }
```
What do you think?